frontend/pages/SoftwarePage/components/icons/TeamViewerHost.tsx

```diff
@@ -1,14 +0,0 @@
-import React from "react";
```

Comment:
this isn't used anywhere and it's named wrong, I think it's an accidental commit from a copy paste